decidim-admin/app/controllers/decidim/admin/concerns/has_private_users_csv_import.rb

```diff
@@ -18,6 +18,7 @@ module HasPrivateUsersCsvImport
           def new
             enforce_permission_to :csv_import, :space_private_user
             @form = form(ParticipatorySpacePrivateUserCsvImportForm).from_params({}, privatable_to: privatable_to)
+            @count = Decidim::ParticipatorySpacePrivateUser.where(privatable_to_id: privatable_to.id, privatable_to_type: privatable_to.class.to_s).count
```

Comment:
I think we can DRY this up a bit. Can you add an Active Record scope with parameter `privatable_to`? The final result would be something like 
```suggestion
            @count = Decidim::ParticipatorySpacePrivateUser.by_privatable(privatable_to).count
```

If you find a better naming for this, that'd be great.